removing documentation in foundry.md and adding link to gitbook docs (#154)
runtimeverification/kontrol: src/kontrol/kdist/foundry.md

#[Kontrol documentation](https://docs.runtimeverification.com/kontrol).

The documentation below may become deprecated. The documentation at the link above will be continuously updated and improved.

Foundry Module for Kontrol

> ---------------------------

Reviewer: ```suggestion
--------------------------
```

Foundry's testing framework provides a series of cheatcodes so that developers can specify what situation they want to test.

> This file describes the Kontrol specification of the Foundry testing framework, which includes the definition of said cheatcodes and what does it mean for a test to pass.

Reviewer: This might be better.

```suggestion
This file describes the K semantics of the Foundry testing framework, which includes the definition of said cheatcodes and what does it mean for a test to pass.
```

      [priority(35)]
```

> The `#halt` production is used to examine the end of each call in Kontrol.

Reviewer: Here, `KEVM` is not used as the tool's name, but as the semantics on top of which Kontrol is defined. Basically, the line says that we're examining the end of each call in `evm-semantics`.

```

Restricting the accounts that can be called in Kontrol

> ---------------------------------------------------

Reviewer: ```suggestion
------------------------------------------------------
```

function allowCallsToAddress(address) external;
```

> Adds an account address to the whitelist. The execution of the modified Kontrol will stop when a call has been made to an address which is not in the whitelist.

Reviewer: Same as above. Maybe you could rewrite as `.. of the modified EVM semantics ..`?

Author: I changed it back to KEVM, same as above. I wasn't sure which KEVMs needed to be changed to Kontrol. SO I really appreciate the review! Thanks so much!
